test/helper.go

```diff
@@ -119,7 +119,7 @@ func NewHelperWithHooks(t *testing.T, httpServer *httptest.Server, configuration
 
 	// Set server if provided
 	if httpServer != nil && ocmConfig.MockMode == ocm.MockModeEmulateServer {
-		workers.RepeatInterval = 1 * time.Second
+		workers.NewReconcilerConfig().RepeatInterval = 1 * time.Second
```

Comment:
We'll need to modify the reconciler value that has been populated and loaded into the test server and not instantiate a new one. 

You could load the reconciler config just like how the other configs are loaded here https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/b094b22b703cb9d70764583947ce0d49ce4bec28/test/helper.go#L103-L107

and in on this line you could re-assign the value of the loaded config 
```go
reconcilerConfig.RepeatInterval=1*time.Second
```

Reply:
Perfect :) thank you